packages/app/src/systems/Sidebar/machines/sidebarMachine.test.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  SideBarActionTypes,
+  sidebarMachine,
+  SideBarStateValues,
+} from './sidebarMachine';
+
+describe('sidebarMachine', () => {
+  it('should open the sideBar', () => {
+    const nextState = sidebarMachine.transition(SideBarStateValues.Minimized, {
+      type: SideBarActionTypes.TOGGLE,
+    });
+    expect(nextState.value).toBe(SideBarStateValues.Expanded);
+  });
+  it('should close the sideBar', () => {
+    const nextState = sidebarMachine.transition(SideBarStateValues.Expanded, {
+      type: SideBarActionTypes.TOGGLE,
+    });
+    expect(nextState.value).toBe(SideBarStateValues.Minimized);
+  });
```

Comment:
I felt the missing click on the item and see the close bar closing. Also E2E tests